rskj-core/src/main/java/co/rsk/signing/SignAuthorization.java

```diff
@@ -0,0 +1,11 @@
+package co.rsk.signing;
+
+/**
+ * Implementors of this should
+ * be able to produce specific
+ * types of signing authorization.
+ *
+ * @author Ariel Mendelzon
+ */
+public interface SignAuthorization {
```

Comment:
This class is not yet used, and no methods are provided. I understand it might be a good design idea, but [we don't need it yet](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it). You can defer this design decision to the moment you actually need a sign authorization.

Reply:
I think this deserves a broader discussion with @juli given that we came up with this interface with him. I agree in that this is not being used at the moment, and had the same doubt. Nevertheless, it's still part of the original design, and should be therefore discussed accordingly.